### PR TITLE
Declaring safeguarding concerns

### PR DESCRIPTION
**Pull request by EmmaFrith**
first draft of entry

screenshots to follow

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Our research was limited due to the difficulty of recruiting candidates who have convictions or cautions, who have been banned from working with children or who have been removed from teacher training in the past due to unprofessional behaviour.
+
+However, we were able to observe a few pain points for candidates. The content lacks sufficient clarity, particularly around:
+
+* what sort of information to declare
+* what the purpose of this section is
+* whether they should be open or not  
+
+### Our hypothesis
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md]:
```suggestion
### Our hypotheses
```

File: app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+1. If we list the types of checks providers carry out
+Then candidates will know whether to share information or not
+Because they'll know what type of information providers are looking for
+
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md:55]: Why the double line breaks between hypotheses?

File: app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+2. If we make it clearer that this section is about safeguarding children and young people
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md]:
```suggestion
2. If we make it clear that this section is about safeguarding children and young people
```

**Review by fofr — COMMENTED** [review]
Words look good, will hold off on approving until screenshots are added.

File: app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md (hunk 2 above)
Comment (EmmaFrith) [reply app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md:55]: oh

**Pull request merged**